indices/build_pairwise_networks.py

```diff
@@ -9,6 +9,7 @@
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
+    parser.add_argument('fragment', help='Which sixteenth of the headings to process', type=int)
```

Comment:
Really it's specific to being able to run quickly enough on the cluster to avoid dying. I might split this into a parallel and nonparallel version at some point in the future because the sixteenths thing is kind of confusing